activerecord/test/cases/base_test.rb

```diff
@@ -1464,4 +1464,9 @@ def test_default_values_are_deeply_dupped
     assert SubDeveloper.new.respond_to?(:last_name=)
     assert SubDeveloper.new.respond_to?(:last_name?)
   end
+
+  test "ignored columns are stored as an array of string" do
+    assert_equal(%w(first_name last_name), Developer.ignored_columns)
```

Comment:
Added this assertion to clearify.

Reply:
Again, can you add more assertions in other three ignored columns tests to ensure that works as expected?

```diff
--- a/activerecord/test/cases/base_test.rb
+++ b/activerecord/test/cases/base_test.rb
@@ -1445,6 +1445,7 @@ def test_default_values_are_deeply_dupped
     assert_includes cache_columns.keys, "first_name"
     assert_not_includes Developer.columns_hash.keys, "first_name"
     assert_not_includes SubDeveloper.columns_hash.keys, "first_name"
+    assert_not_includes SymbolIgnoredDeveloper.columns_hash.keys, "first_name"
   end
 
   test "ignored columns have no attribute methods" do
@@ -1454,6 +1455,9 @@ def test_default_values_are_deeply_dupped
     refute SubDeveloper.new.respond_to?(:first_name)
     refute SubDeveloper.new.respond_to?(:first_name=)
     refute SubDeveloper.new.respond_to?(:first_name?)
+    refute SymbolIgnoredDeveloper.new.respond_to?(:first_name)
+    refute SymbolIgnoredDeveloper.new.respond_to?(:first_name=)
+    refute SymbolIgnoredDeveloper.new.respond_to?(:first_name?)
   end
 
   test "ignored columns don't prevent explicit declaration of attribute methods" do
@@ -1463,6 +1467,9 @@ def test_default_values_are_deeply_dupped
     assert SubDeveloper.new.respond_to?(:last_name)
     assert SubDeveloper.new.respond_to?(:last_name=)
     assert SubDeveloper.new.respond_to?(:last_name?)
+    assert SymbolIgnoredDeveloper.new.respond_to?(:last_name)
+    assert SymbolIgnoredDeveloper.new.respond_to?(:last_name=)
+    assert SymbolIgnoredDeveloper.new.respond_to?(:last_name?)
   end
 
   test "ignored columns are stored as an array of string" do
```

Reply:
Thank you. I applied the changes.